gateway/1.4/modules/ROOT/pages/_partials/forward-proxy-config.adoc

```diff
@@ -0,0 +1,164 @@
+//tag::configuration[]
+A forward proxy, such as Squid, enables you to route external HTTP Flex Gateway connections through a proxy connection.  
+
+For both Connected Mode and Local Mode, you configure a forward proxy using YAML configuration files.
+
+The following diagram demonstrates how an internal network communicates with an external network using a forward proxy. 
+
+image:forward-proxy-diagram.png[align=center]
+
+As in the diagram, when using a forward proxy, HTTP communication out of your internal network must travel through the forward proxy. 
+
+In the previous example, all HTTP API upstream communication must also pass through the forward proxy. For more information about how to enable some connections to communicate directly, without the forward proxy, see <<noproxy-parameter, noProxy Parameter>>.
+
+== Forward Proxy Connections
+Only HTTP connections route through the forward proxy.
+
+The following connections route through the forward proxy:
+
+* Connections to Anypoint Platform
+* Outbound policy connections
+* Connections to upstream services
+* HTTP (Fluent Bit) log connections
+
+Using the `noProxy` parameter you can configure HTTP and HTTPS connections to connect directly without using the forward proxy.
+
+Configuring a forward proxy does not alter these connections:
+
+* Connections to shared storage
+* Lightweight Directory Access Protocol (LDAP) connections
+* Non-HTTP log connections
+
+== noProxy Parameter
+
+Adding the `noProxy` parameter allows HTTP services you trust to communicate without using the forward proxy.
+
+For example, when you enable `noProxy`, you can list connections to upstream services and HTTP log connections to connect directly to Flex Gateway. 
```

Comment:
This wording is a bit off. I suggest this instead (if it is accurate):

For example, when you enable `noProxy`, you can list HTTP log connections and connections to upstream services to have them connect directly to Flex Gateway.